src/components/LoadableComponentErrorBoundary/LoadableComponentErrorBoundaryPage.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { FormattedMessage } from '../../util/reactIntl';
+import { pathByRouteName } from '../../util/routes';
+import routeConfiguration from '../../routeConfiguration';
+import {
+  LayoutSingleColumn,
+  LayoutWrapperTopbar,
+  LayoutWrapperMain,
+  InlineTextButton,
+  Logo,
+} from '../../components';
+
+import css from './LoadableComponentErrorBoundary.module.css';
+
+export const LoadableComponentErrorBoundaryPage = () => {
+  const landingPagePath = pathByRouteName('LandingPage', routeConfiguration());
+  const handleOnClick = () => {
+    if (typeof window !== 'undefined') {
+      window.location = landingPagePath;
+    }
+  };
+
+  const landingPageLink = (
+    <InlineTextButton onClick={handleOnClick}>
+      <FormattedMessage id="LoadableComponentErrorBoundaryPage.landingPageLink" />
+    </InlineTextButton>
+  );
+
+  const topbar = (
```

Comment:
I think you can move this code inside LayoutWrapperTopbar - no need to keep it in different variable.